test_private_network.py

```diff
@@ -142,33 +142,75 @@ def get_interface_config(server_index):
         s1.ping(f'192.168.{octet}.2', tries=10, wait=1)
 
 
-def test_no_private_network_port_security(create_server, image, server_group):
+def test_no_private_network_port_security(
+    create_server,
+    image,
+    server_group,
+    private_network
+):
     """ Private networks can do what public networks can't:
 
     * Change the mac address of an interface.
     * Change the IP address of an interface.
 
     """
 
+    # Define a custom subnet
+    subnet = private_network.add_subnet(cidr='192.168.100.0/24')
+
+    s1_address = '192.168.100.1'
+    s2_address = '192.168.100.1'
+
+    # Initialize each server with an interface attached to the private network.
+    # The servers are not given an address by DHCP
```

Comment:
I think the way we write this in the API docs is a tad clearer:

> No IP address will be assigned using DHCP

https://www.cloudscale.ch/en/api/v1#interfaces-attribute-specification

Reply:
Updated with text from API.